Resources/Prototypes/_LostParadise/Loadouts/Jobs/security.yml

```
@@ -122,17 +852,9 @@
-  requirements:
-    - !type:CharacterItemGroupRequirement
-      group: LoadoutBeltSecurity
-    - !type:CharacterDepartmentRequirement
-      departments:
-        - Security
+  id: LoadoutItemWeaponLauncherChinaLakeMeme
+  category: Items
+  cost: 999
   items:
-    - ClothingBeltHolster
+    - WeaponLauncherChinaLake
```

yamllint

[error] no new line character at the end of file

(new-line-at-end-of-file)